src/analysis/retail/druid/restoration/CONFIG.tsx

```diff
@@ -3,7 +3,7 @@ import Expansion from 'game/Expansion';
 import SPECS from 'game/SPECS';
 import Config from 'parser/Config';
 
-import CHANGELOG from './CHANGELOG';
+// import CHANGELOG from './CHANGELOG';
```

Comment:
what happened here

Reply:
merge conflict/rebase conflict if I had to guess